exercises/nth-prime/src/example.c

```diff
@@ -0,0 +1,32 @@
+#include "nth_prime.h"
+#include <stdbool.h>
+#include <math.h>
+
+static bool is_prime(uint32_t n)
+{
+  for(uint32_t i = (uint32_t)sqrt(n); i > 1; i--) {
```

Comment:
Yeah my bad. I was thinking a sieve could be used to find N primes, but I now realize there's no good way to set the upper bound on the sieve in order to guarantee that you'll find N primes.